docs/src/main/sphinx/release/release-403.md

```diff
@@ -0,0 +1,62 @@
+# Release 403 (12 Nov 2022)
+
+## General
+
+* Add the amount of data read from external sources during table scans to
+  `EXPLAIN ANALYZE`. ({issue}`14907`)
+* Fix handling of `time(n) with time zone` when clients do not support variable
+  precision temporal types. ({issue}`14950`)
+* Fix query deadlock in multi-join queries where broadcast join size is
+  underestimated. ({issue}`14948`)
+* Fix incorrect results when `min(x, n)` or `max(x, n)` is used as a window
+  function. ({issue}`14886`)
+* Fix failure for certain queries involving joins over partitioned tables. ({issue}`14317`)
+* Fix incorrect order of parameters in `DESCRIBE INPUT` when they appear in a
+  `WITH` clause. ({issue}`14738`)
+
+## Security
+
+* Allow access token passthrough when using OAuth 2.0 with refresh tokens
+  enabled. ({issue}`14949`)
+
+## Fault-tolerant execution
+
+* Override the values of the `exchange.compression-enabled`,
```

Comment:
Override -> Automatically set/configure?

Reply:
I think override is more clear for the sake of establishing that even if you otherwise have configuration values for these properties, they're being ignored when you turn on FTE.

Reply:
We’re going to have to fix that. Silently overriding the values is error prone. It also goes against the principles of how airlift configuration works - when there are unused properties or usage of defunct properties, the server should refuse to start. cc @arhimondr

Reply:
It doesn't silently override. It's using `bindConfigDefaults` which only change default values - not any user specified values.

Reply:
In that case, I don't think we even need to document this. If the user is already setting these explicitly, their settings will be preserved. If they are not, they get handled automatically.

Reply:
Yeah, let's not document it. We should reflect these changes on the fault tolerant execution configuration guide though, so we no longer suggest users to set these properties explicitly.

Reply:
By "not document" do we mean not include a release note at all?

Reply:
Correct